### PR TITLE
feat: Create script to sort table by columns outside mySQL

### PR DESCRIPTION
**Pull request by diegomscoelho**
## Description

During dbSNP import sorting 1 billion row table seems to be a slow process. To enhance this task we might use a script that:
* Dump entire table;
* Sort using desired columns;
* Load entire table back into database;

This strategy were able to sort 1-billion rows table in less than 2 days, which is better if compared to +14 days using "ALTER TABLE ... ORDER BY ...".

## Test

1) Create a database with a table like variation_feature;
2) Unsort table (you might order using variation_id);
3) Run script sorting another desired columns;

### CODE REVIEW

File: scripts/import/dbSNP_v2/sort_table_by_columns.pl [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+$dbh->do( "LOAD DATA LOCAL INFILE \"${TMP_DIR}/sorted_${TMP_FILE}\" INTO TABLE ${table_name}_tmp") or die "Failed to load infile in tmp table";
+$dbh->do( "ALTER TABLE ${table_name}_tmp ENABLE KEYS") or die "Failed to re-enable keys in tmp table";
+
+system("rm ${TMP_DIR}/*${TMP_FILE}*");
+
+warn("TIP!!! REMEMBER TO EXECUTE SQL IN SERVER:\nDROP TABLE ${dbname}.${table_name};\nRENAME TABLE ${dbname}.${table_name}_tmp TO ${dbname}.${table_name};\n");
+
+sub usage{
+
+  die "\n\tUsage: update_changed_sets.pl
```
Comment (dglemos) [inline scripts/import/dbSNP_v2/sort_table_by_columns.pl:89]: Update the script name to `sort_table_by_columns.pl`

Comment (diegomscoelho) [reply scripts/import/dbSNP_v2/sort_table_by_columns.pl]: Done.

File: scripts/import/dbSNP_v2/sort_table_by_columns.pl [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+system("rm ${TMP_DIR}/*${TMP_FILE}*");
+
+warn("TIP!!! REMEMBER TO EXECUTE SQL IN SERVER:\nDROP TABLE ${dbname}.${table_name};\nRENAME TABLE ${dbname}.${table_name}_tmp TO ${dbname}.${table_name};\n");
+
+sub usage{
+
+  die "\n\tUsage: sort_table_by_cols.pl
+  \t\t-registry [registry file]
+  \t\t-columns [comma separated numbers]
```
[the remark this answers is not in the document]
Comment (diegomscoelho) [reply scripts/import/dbSNP_v2/sort_table_by_columns.pl]: Done.

File: scripts/import/dbSNP_v2/sort_table_by_columns.pl [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+system("sort --parallel=8 --buffer-size=30G ${cols_string}-o $TMP_DIR/sorted_$TMP_FILE $TMP_DIR/$TMP_FILE");
+$dbh->do( "DROP TABLE IF EXISTS ${table_name}_tmp" ) or die "Failed to drop pre-existing temp table";
+$dbh->do( "CREATE TABLE ${table_name}_tmp LIKE $table_name") or die "Failed to create temp table";
+$dbh->do( "ALTER TABLE ${table_name}_tmp DISABLE KEYS") or die "Failed to disable keys in tmp table";
+$dbh->do( "LOAD DATA LOCAL INFILE \"${TMP_DIR}/sorted_${TMP_FILE}\" INTO TABLE ${table_name}_tmp") or die "Failed to load infile in tmp table";
+$dbh->do( "ALTER TABLE ${table_name}_tmp ENABLE KEYS") or die "Failed to re-enable keys in tmp table";
+
+system("rm ${TMP_DIR}/*${TMP_FILE}*");
+
+warn("TIP!!! REMEMBER TO EXECUTE SQL IN SERVER:\nDROP TABLE ${dbname}.${table_name};\nRENAME TABLE ${dbname}.${table_name}_tmp TO ${dbname}.${table_name};\n");
```
[the remark this answers is not in the document]
Comment (diegomscoelho) [reply scripts/import/dbSNP_v2/sort_table_by_columns.pl]: Done.

**Review by dglemos — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/109 and main

**Pull request merged**